cloudstate-kotlin-support/pom.xml

```diff
@@ -82,7 +94,9 @@
     </dependencies>
 
     <build>
-        <sourceDirectory>${project.basedir}/src/main/kotlin</sourceDirectory>
+        <!--<sourceDirectory>${project.basedir}/src/main/java</sourceDirectory>
```

Comment:
Intentionally left here?

Reply:
Hi @viktorklang  in another PR (#7 ) i already fixed this